### DISCUSSION TITLE
Remove timeout

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#7271] [Q&A] [answered]
Please i need to remove the timeout for sessions that ask me to login again. Any help please ?

**Comment by jasonvarga** [comment, marked as answer]
Increase session lifetime.

https://github.com/statamic/statamic/blob/da2f408fb219534b707481b04c33c4e6acea0fe0/config/session.php#L34

**Reply by HazemMohamed252** [reply]
Thanks
